### PR TITLE
Bring Facebook\Http src and tests into PSR-2 compliance

### PR DESCRIPTION
**Pull request by rpalladino**
This brings the Facebook\Http namespace plus its tests into PSR-2 compliance

### CODE REVIEW

**Comment by SammyK** [comment]
:+1:

File: src/Facebook/Http/RequestBodyMultipart.php [hunk 1]
```diff
@@ -37,117 +37,116 @@
...
+    /**
+     * @param array  $params   The parameters to send with this request.
+     * @param array  $files    The files to send with this request.
+     * @param string $boundary Provide a specific boundary.
+     */
+    public function __construct(
+        array $params = [],
+        array $files = [],
+        $boundary = null
+    ) {
```
Comment (yguedidi) [inline src/Facebook/Http/RequestBodyMultipart.php]: One line IMO

Comment (yguedidi) [inline src/Facebook/Http/RequestBodyMultipart.php]: And the brace should be in its own line

File: tests/Http/GraphRawResponseTest.php [hunk 2]
```diff
@@ -28,54 +28,53 @@
 class GraphRawResponseTest extends \PHPUnit_Framework_TestCase
 {
 
-  protected $fakeRawProxyHeader = "HTTP/1.0 200 Connection established
+    protected $fakeRawProxyHeader = "HTTP/1.0 200 Connection established
 Proxy-agent: Kerio Control/7.1.1 build 1971\r\n\r\n";
-  protected $fakeRawHeader = "HTTP/1.1 200 OK
+    protected $fakeRawHeader = "HTTP/1.1 200 OK
```
Comment (yguedidi) [inline tests/Http/GraphRawResponseTest.php]: I think a HEREDOC here is clearer

File: tests/Http/RequestUrlEncodedTest.php [hunk 3]
```diff
@@ -28,15 +28,14 @@
...
-    $body = $message->getBody();
-
-    $this->assertEquals('foo=bar&scawy_vawues=%40FooBar+is+a+real+twitter+handle.', $body);
-  }
+    public function testCanProperlyEncodeAnArrayOfParams()
+    {
+        $message = new RequestBodyUrlEncoded([
+                'foo' => 'bar',
+                'scawy_vawues' => '@FooBar is a real twitter handle.',
+            ]);
```
Comment (yguedidi) [inline tests/Http/RequestUrlEncodedTest.php]: You should remove one indentation in the 3 above lines

**Comment by yguedidi** [comment]
Please see #356

**Comment by gfosco** [comment]
Closing due to #356.  If this is still valid after #356 please resubmit then.

**Pull request closed without merging**